src/DeployNuGetPackage/DeployNuGetPackageV2/src/DeployNuGetPackage.ts

```diff
@@ -44,7 +44,7 @@ export async function run() {
         "-SourceApp", SourceApp,
         "-DirectDeploy", DirectDeploy,
         "-UseMaintenancePage", UseMaintenancePage,
-        "-DropPath", DropPath,
+        "-DropPath", `"${DropPath}"`,
```

Comment:
@ovelartelius these pages suggest that single quotes might work in both windows and ubuntu.

https://medium.com/@leedowthwaite/dealing-with-spaces-in-paths-f26856aef06f
https://www.delftstack.com/howto/powershell/powershell-path-with-spaces/

`"-DropPath", `'${DropPath}'`,`